features/fixtures/mazerunner/cxx-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/CXXDelayedCrashScenario.kt

```diff
@@ -26,7 +27,9 @@ class CXXDelayedCrashScenario(
 
     override fun onActivityStopped(activity: Activity) {
         super.onActivityStopped(activity)
-        log("App sent to background, triggering crash.")
-        activate(405)
+        log("App sent to background, triggering crash in 1 seconds.")
+        Handler().postDelayed({
+            activate(405)
+        }, 1000)
```

Comment:
This makes logical sense to me, but it feels like there's something else going on here.  I ran it in a loop 22 times and it failed 6 times, but increasing the delay to 3s made it worse.

Perhaps we should be scrutinising the internals of the notifier more and adding some logging (as I think you were suggesting IRL)?

Reply:
Maybe we need some more logging? In say `onActivityResumed` as well (or possibly even *all* of the activity lifecycle methods)?